css/style.css

```diff
@@ -0,0 +1,134 @@
+body {
```

Comment:
Remember you generally don't need to use tag selectors.

Reply:
Edited to .class